merge_requests.go

```diff
@@ -511,6 +511,58 @@ func (s *MergeRequestsService) CancelMergeWhenPipelineSucceeds(pid interface{},
 	return m, resp, err
 }
 
+// Subscribe subscribes the authenticated user to the given merge request to receive notifications.
+// If the user is already subscribed to the merge request, the status code 304 is returned.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/merge_requests.html#subscribe-to-a-merge-request
+func (s *MergeRequestsService) Subscribe(pid interface{}, mergeRequest int, options ...OptionFunc) (*MergeRequest, *Response, error) {
```

Comment:
I know it's a bit longer, but it feels more consistent to make this `SubscribeToMergeRequest`.

I understand that a lot of these methods could maybe be shorter because they are already defined on the `MergeRequestService`, but since I once started with names that mimic how the API call is named in the docs, I would like to keep doing that.